packages/context/src/context.ts

```diff
@@ -11,8 +11,9 @@ export type Constructor<T> = new(...args) => T;
 export class Context {
   private registry: Map<string, Binding>;
 
-  constructor() {
+  constructor(private _parent?: Context) {
     this.registry = new Map();
```

Comment:
I think `uuid` is good enough. We should also check out http://opentracing.io/documentation/.